scss/_patterns_search-box.scss

```diff
@@ -124,16 +122,20 @@
 
 @mixin vf-search-box-light-theme {
   @include vf-search-box-theme(
-    $color-search-box-background: $colors--light-theme--background-default,
+    $color-search-box-background: $colors--light-theme--background-inputs,
     $color-search-box-border: $colors--light-theme--border-high-contrast,
     $color-search-box-text: $colors--light-theme--text-default
   );
 }
 
 @mixin vf-search-box-dark-theme {
   @include vf-search-box-theme(
-    $color-search-box-background: $colors--dark-theme--background-default,
-    $color-search-box-border: $colors--dark-theme--border-high-contrast,
+    $color-search-box-background: lighten($colors--dark-theme--background-default, 10%),
+    $color-search-box-border: $color-x-light,
     $color-search-box-text: $colors--dark-theme--text-default
   );
+
+  &::placeholder {
```

Comment:
This should not be here. This should only contain calling the theme with specific colors.

If some color needs to be set on placeholder it should go inside the `vf-search-box-theme`

Reply:
But we want that only for the dark theme version, wouldn't it make sense to have it in `vf-search-box-dark-theme` ?

Reply:
Technically yes, but I don't want to mess up with the convention too much. To make things consistent all colours should be controlled by the theme mixin.

So in this case, if we want to change the placeholder color, it would need to be added as another parameter to the `vf-search-box-theme` mixin, and correct values for light/dark would be passed.

But also, should the placeholder color be the same as the text? @lyubomir-popov

Reply:
no, it should be the lightest possible that still passes contrast ratio reqs. The varialbes are `$colors--light-theme--text-muted` and `$colors--dark-theme--text-muted`. It appears we're defaulting to whatever the browser decides at the moment as I can't find `::placeholder` styles.